src/borg/archiver.py

```diff
@@ -726,8 +726,8 @@ def do_mount(self, args, repository, manifest, key):
             self.print_error('Loading fuse support failed [ImportError: %s]' % str(e))
             return self.exit_code
 
-        if not os.path.isdir(args.mountpoint) or not os.access(args.mountpoint, os.R_OK | os.W_OK | os.X_OK):
-            self.print_error('%s: Mountpoint must be a writable directory' % args.mountpoint)
```

Comment:
I see.
Ok, I did not know fuse was supposed to be run by any user in the fuse group, so they needed to guard against users mounting stuff on top of /etc and so on. It makes sense then.